lib/stack/NodeBuildInterpreter.ts

```diff
@@ -73,6 +77,13 @@ export class NodeBuildInterpreter implements Interpreter, AutofixRegisteringInte
             action: runFingerprints(fingerprintRunner([NpmDependencyFingerprint])),
         });
 
+    private readonly dockerBuildGoal: DockerBuild = new DockerBuild()
+        .with({
```

Comment:
this seems to be missing the progress reporter and the docker file finder from uhura.

Reply:
I will add it